src/main/java/net/fabricmc/loader/impl/metadata/V1ModMetadataParser.java

```diff
@@ -382,13 +385,14 @@ private static void readMixinConfigs(List<ParseWarning> warnings, JsonReader rea
 			switch (reader.peek()) {
 			case STRING:
 				// All mixin configs specified via string are assumed to be universal
-				mixins.add(new V1ModMetadata.MixinEntry(reader.nextString(), ModEnvironment.UNIVERSAL));
+				mixins.add(new V1ModMetadata.MixinEntry(reader.nextString(), ModEnvironment.UNIVERSAL, Collections.emptySet()));
 				break;
 			case BEGIN_OBJECT:
 				reader.beginObject();
 
 				String config = null;
 				ModEnvironment environment = null;
+				Set<String> depends = null;
```

Comment:
Shouldn't this default to an empty set like before rather than null?

Reply:
Nevermind it's set later, not sure why it isn't just initialised here though.

Reply:
It is a little odd, I was just following the same pattern elsewhere in the class (including `environment`)